config/resources/organisation.lisp

```diff
@@ -1,11 +1,42 @@
+(define-resource organization ()
+  :class (s-prefix "org:Organization")
+  :properties `((:name :string ,(s-prefix "skos:prefLabel")))
+  :has-one `(
+            (organization-classification-codes :via ,(s-prefix "org:classification")
```

Comment:
```suggestion
            (organization-classification-code :via ,(s-prefix "org:classification")
```
Relates to https://github.com/lblod/app-subsidiepunt/pull/22/files#r1686168881

Reply:
has-one = singular